src/applications/simple-forms/20-10207/pages/nonVeteranPhoneAndEmail.js

```diff
@@ -1,59 +1,37 @@
 import {
-  emailSchema,
-  emailUI,
   emailToSendNotificationsSchema,
   emailToSendNotificationsUI,
   phoneSchema,
   phoneUI,
   titleUI,
 } from 'platform/forms-system/src/js/web-component-patterns';
-import environment from 'platform/utilities/environment';
 import {
   getPhoneAndEmailPageEmailHint,
   getPhoneAndEmailPageTitle,
 } from '../helpers';
 
 /** @type {PageSchema} */
-// eslint-disable-next-line import/no-mutable-exports
-let pageSchema = {
+const pageSchema = {
```

Comment:
Unfortunately github is confused on the diff - it's much more straight forward than this.
The changes are removing the top block, and keeping the bottom block for each file.